app/services/notification/AWSMailClient.scala

```diff
@@ -0,0 +1,46 @@
+package services.notification
+
+import javax.inject.Inject
+
+import com.amazonaws.handlers.AsyncHandler
+import com.amazonaws.services.simpleemail.AmazonSimpleEmailServiceAsync
+import com.amazonaws.services.simpleemail.model._
+import models.{Instance, Owner}
+import play.api.libs.concurrent.Execution.Implicits._
+import play.api.{Configuration, Logger}
+
+import scala.concurrent.{Future, Promise}
+
+class AWSMailClient @Inject()(amazonMailClient: AmazonSimpleEmailServiceAsync, configuration: Configuration) {
+
+  def send(owner: Owner, instances: Seq[Instance]): Future[String] = {
```

Comment:
Am I right to think this will send even if there are no old instances to mention? Is this the expected behaviour? This should probably just do as its told, I think the logic for that should probably be elsewhere but I spotted it here so I'll mention it here.